08_deploy_bmo.sh

```diff
@@ -5,12 +5,13 @@ eval "$(go env)"
 # Get the latest bits for baremetal-operator
 export BMOPATH="$GOPATH/src/github.com/metalkube/baremetal-operator"
 
-oc --as system:admin --config ocp/auth/kubeconfig new-project bmo-project
+# Make a local copy of the baremetal-operator code to make changes
+cp -r $BMOPATH/deploy .
+sed -i 's/bmo-project/openshift-machine-api/g' deploy/role_binding.yaml
 
 # Start deploying on the new cluster
-
-oc --as system:admin --config ocp/auth/kubeconfig apply -f $BMOPATH/deploy/service_account.yaml
-oc --as system:admin --config ocp/auth/kubeconfig apply -f $BMOPATH/deploy/role.yaml
-oc --as system:admin --config ocp/auth/kubeconfig apply -f $BMOPATH/deploy/role_binding.yaml
-oc --as system:admin --config ocp/auth/kubeconfig apply -f $BMOPATH/deploy/crds/metalkube_v1alpha1_baremetalhost_crd.yaml
-oc --as system:admin --config ocp/auth/kubeconfig apply -f $BMOPATH/deploy/operator.yaml
+oc --as system:admin --config ocp/auth/kubeconfig apply -f deploy/service_account.yaml
+oc --as system:admin --config ocp/auth/kubeconfig apply -f deploy/role.yaml
+oc --as system:admin --config ocp/auth/kubeconfig apply -f deploy/role_binding.yaml
+oc --as system:admin --config ocp/auth/kubeconfig apply -f deploy/crds/metalkube_v1alpha1_baremetalhost_crd.yaml
+oc --as system:admin --config ocp/auth/kubeconfig apply -f deploy/operator.yaml
```

Comment:
We need to make sure this uses the openshift-machine-api namespace.

Reply:
The change in line #10 is updating the namespace to openshift-machine-api in the role_binding.yaml file. Do you think it needs to be changed in other places? Other .yaml files do not explicitly have a namespace field. Do you want me to add the namespace field explicitly to the metadata section of the other (other than role_binding.yaml) .yaml files?